### ISSUE TITLE
Double Lock Sign in Voice Channels

### ISSUE THREAD

**Issue by KevGOtt** [JustOptimize/return-ShowHiddenChannels#48]
https://prnt.sc/A-4YRdAhbkj7

**Comment by Ridoou** [comment]
Yep, same thing for me it's since the 0.0.3 update who add lock so it's normal but weird to have 2

**Comment by JustOptimize** [comment]
It's not a bug, I will add a setting to move it or disable it entirely in the future and maybe change the emoji

**Comment by aVitomin** [comment]
> It's not a bug, I will add a setting to move it or disable it entirely in the future and maybe change the emoji
Disable emoji will be great

**Comment by KingCharlesVI** [comment]
Try changing line no. 226

From:
`                returnValue.name = " 🔒 " + returnValue.name;`

To:
`                returnValue.name = returnValue.name;`

I'm no JS exper and this probably won't work, but logically it might.

**Comment by dimaguy** [comment]
> Try changing line no. 226
> 
> From:
> `                returnValue.name = " 🔒 " + returnValue.name;`
> 
> To:
> `                returnValue.name = returnValue.name;`
> 
> I'm no JS expert and this probably won't work, but logically it might.

While it might work, editing manually the source for each upcoming update would be a chore, so I wouldn't recommend this on the long run

**Comment by JustOptimize** [comment]
> > Try changing line no. 226
> > From:
> > `                returnValue.name = " 🔒 " + returnValue.name;`
> > To:
> > `                returnValue.name = returnValue.name;`
> > I'm no JS expert and this probably won't work, but logically it might.
> 
> While it might work, editing manually the source for each upcoming update would be a chore, so I wouldn't recommend this on the long run

If you want to remove it just delete that lines 🤦🏻‍♂️ (From the `var channelCha...` to `});`)

**Comment by JustOptimize** [comment]
Check the latest update, I added one of the settings you requested
PS: Do not close this issue because the ability to choose which emoji to use will soon be available.

**Comment by JustOptimize** [comment]
Is everything to your liking? [Download](https://github.com/JustOptimize/return-ShowHiddenChannels/blob/dev-unstable/ShowHiddenChannels.plugin.js)

**Comment by black-trush** [comment]
Is it possible to replace the # with a padlock?

**Comment by Tomo-Tg** [comment]
I hope it will look like the image above.
Frankly, I don't like the presence of colored emojis in the channel list ... it's not uniform

**Comment by KevGOtt** [comment]
I kinda also want it to look like the image above.

**Comment by black-trush** [comment]
> I hope it will look like the image above. Frankly, I don't like the presence of colored emojis in the channel list ... it's not uniform
> I kinda also want it to look like the image above. Maybe add that you cant click locked Channels.

that image I make it. that's for example.
i think so, replace a not permissoin text channels to not permission voice channels.

**Comment by JustOptimize** [comment]
Will this be enough to close this issue? 
![image](https://user-images.githubusercontent.com/54294419/198358439-b0558094-aec0-4d3b-bd03-20d46dd05bbd.png)

**Comment by KevGOtt** [comment]
Can you move the lock to the left?

**Comment by JustOptimize** [comment]
No
